content/articles/blockchain-offered-as-a-service/index.md

```diff
@@ -0,0 +1,69 @@
+### Prerequisites
+To understand this article one should have:
+1. Good knowledge and understanding of how blockchain works.
+2. Good understanding of the basics of Blockchain.
+3. Basic information about applications of Blockchain.
+
+### Understanding  BaaS.
+In a world of distributed applications, blockchain can provide to any type of application, including those centralized ones we’re all used to. Nowadays there are many types of blockchains each with its own characteristics which make them more or less suitable for use by large enterprises and small businesses alike. As such, cloud providers should embrace the technology and offer blockchain as a service to their customers.
+
+Today, the technology behind blockchains is still rather complicated for most people to use regularly. For example, setting up a validator node for some public networks such as Ethereum can be a tedious process requiring some Linux knowledge. A large amount of time and effort must be spent to ensure the blockchain software has been configured correctly and is running smoothly without any unforeseen issues. If something goes wrong, one must troubleshoot the issue or look to the documentation for guidance.
+
+The same applies to some other public blockchains such as Bitcoin where full nodes require a server farm to verify transactions since it’s not possible for a single desktop machine to do the job. For enterprises, running a full node for each blockchain they want to use is often not feasible due to financial and resource constraints especially during the early phases of adoption. As such, it would be best if cloud providers started offering blockchain as a service where customers can quickly get up and running with their own nodes or a preconfigured virtual machine.
+### Reasons Companies choose BaaS?
+  1. Businesses that use Blockchain as a Service acquire access to qualified Blockchain developers, procedure and management specialists, the whole infrastructure from cloud service provider for developing, programming, and deploying, not worrying about budget costs for any other operation that would cost them more.
+		
+  2. In most situations, a reputable BaaS partner is a source of practical expertise and knowledge that can improve the security of the systems. It considerably reduces the dangerous cases that might otherwise be encountered if it were created in-house
+
+### Example of application
+A good example of a cloud provider that offers blockchain as a service is Microsoft Azure, with options to use both Ethereum and BitCoin’s blockchains. The service costs about $13 per month for the most basic of configurations which is rather steep compared to some other VPS providers such as DigitalOcean who charge $5.
```

Comment:
Looks like Azure Blockchain services has been stopped since Sept 2021, and was replaced by Consensys Quorum. 

> Azure Blockchain Service was retired on 10th September, 2021. Customers are recommended to use ConsenSys Quorum Blockchain Service as a direct replacement, which offers expanded features and more flexible deployment options for managed blockchain networks.

https://azure.microsoft.com/en-in/solutions/blockchain/#related-products